README.md

```diff
@@ -26,14 +26,25 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] Mention two parts of Express that you learned about this week.
 
+    Routers and Middleware
+
 - [ ] Describe Middleware?
 
+    Just a function! It's the code that is ran on an endpoint or globablly on all endpoint before the code inside the endpoint. Think of an assembly line that performs checks or adds new parts to an item.
+
 - [ ] Describe a Resource?
 
+      A resource is the thing being set back to the client after their request.
```

Comment:
resources are the noun of your application. (shoes / locations / dogs / actions  etc etc). Grouping / Category of data